### ISSUE TITLE
[Bug]: Admin Settings: Unable to edit the Translation section for receivers

### ISSUE THREAD

**Issue by sethdarragile6** [CDCgov/prime-reportstream#4293]
### What happened?

From Jorge: 
"Unable to edit the translation section. I was unable to type in the box, but I was able to use backspace to delete."

At the time of the bug bash, I was able to repro this but am unable to now.

![image](https://user-images.githubusercontent.com/92405130/153520177-593e8e0c-d1f7-47ba-bd7a-065aff3e7773.png)

### Version

1.0.2 (Default)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sethdarragile6** [comment]
hey gang! I believe I had made a quick fix since @jorg3lopez found this bug at the Bug Bash, and thus it's not reproducible. However I used this opportunity to better drill into some of the other general weirdness that was encountered on similar JSON textareas, and found that 1) **wow**, I'd really happy-pathed this Receiver edit screen and 2) each of these fields seem to have their own little built-in rules/behavior about what's valid and what's not from the API perspective:

- "translation" straight up can not be null or empty- period.
- all the filters ("jurisdictionalFilter","qualityFilter","routingFilter","processingModeFilter") must have brackets ([ ])- either with valid list items, or simply "[]" to denote that it's blank. Sending in "" or null to indicate blank is invalid
- "timing" must be "{ _with some valid json_}" or **null** to denote that it's blank. Sending in just "{}" passes validation, but it does NOT blank out the field
- "transport" is like the "timing" field, except that sending in "{}" FAILS validation

I haven't drilled into the top-level org filter or the (upcoming) sender "keys" fields, but I'm guessing that they will have their own peccadillos. I'm just detailing this right now to start a conversation more than anything, but generally this is _all_ a bit problematic to account for on the React side of things. It would be nice if all these types of fields were treated consistently. Thoughts?

@MauriceReeves-usds @TomNUSDS @jimduff-usds @meckila
